src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_ev_driver_windows.cc

```diff
@@ -746,7 +746,7 @@ class SockToPolledFdMap {
           protocol);
       return s;
     }
-    grpc_tcp_set_non_block(s);
+    std::ignore = grpc_tcp_set_non_block(s);
```

Comment:
Seems like the original code missed an error check. We could change it to something like:
```
    grpc_error_handle error = grpc_tcp_set_non_block(s);
    if (!error.ok()) {
      GRPC_CARES_TRACE_LOG("WSAIoctl failed with error: %s",
                           StatusToString(error).c_str());
      return INVALID_SOCKET;
    }
```
@drfloob (who recently worked on Windows platform)  WDYT?

Reply:
SGTM, looks like an oversight.